openapi-codegen/core/src/test/scala/sttp/tapir/codegen/ClassDefinitionGeneratorSpec.scala

```diff
@@ -271,21 +282,19 @@ class ClassDefinitionGeneratorSpec extends CompileCheckTestBase {
     )
 
     val gen = new ClassDefinitionGenerator()
-    val res = gen.classDefs(doc, true, jsonParamRefs = Set("Test"))
-    val resWithQueryParamCodec = gen.classDefs(doc, true, queryParamRefs = Set("Test"), jsonParamRefs = Set("Test"))
+    val res = gen.classDefs(doc, true, jsonParamRefs = Set("Test")).map(_.classRepr)
+    val resWithQueryParamCodec = gen.classDefs(doc, true, queryParamRefs = Set("Test"), jsonParamRefs = Set("Test")).map(_.classRepr)
     // can't just check whether these compile, because our tests only run on scala 2.12 - so instead just eyeball it...
     res shouldBe Some("""
       |
       |enum Test derives org.latestbit.circe.adt.codec.JsonTaggedAdt.PureCodec {
       |  case enum1, enum2
-      |}""".stripMargin)
-    resWithQueryParamCodec shouldBe Some("""
-      |
-      |def enumMap[E: enumextensions.EnumMirror]: Map[String, E] =
+      |}
```

Comment:
I know this test is fragile and sucks a bit because of its intolerance to newlines, but I don't want to fix it in this pr, which is already kinda a lot for me to keep in my head. Sorry. Plz just ignore it for now, I'll get around to it separately.